bazel/linux/kunit/BUILD.bazel

```diff
@@ -1,25 +1,58 @@
 py_binary(
     name = "kunit",
     srcs = ["kunit.py"],
-    visibility = ["//visibility:public"],
     deps = [
-        ":kunit_config",
+        ":kunit_json",
         ":kunit_kernel",
         ":kunit_parser",
     ],
 )
 
 py_library(
-    name = "kunit_config",
-    srcs = ["kunit_config.py"],
+    name = "kunit_json",
+    srcs = ["kunit_json.py"],
+    deps = [
+        ":kunit_parser",
+    ],
 )
 
 py_library(
     name = "kunit_kernel",
     srcs = ["kunit_kernel.py"],
+    deps = [
+        ":kunit_config",
+        ":kunit_parser",
+    ],
 )
 
 py_library(
     name = "kunit_parser",
     srcs = ["kunit_parser.py"],
 )
+
+py_library(
+    name = "kunit_config",
+    srcs = ["kunit_config.py"],
+)
+
+# TODO: This test requires testdata that is not checked in. Check in testdata,
+# fix the test, uncomment this target.
+#py_test(
+#    name = "kunit_test",
+#    srcs = ["kunit_tool_test.py"],
+#    main = "kunit_tool_test.py",
+#    deps = [
+#        ":kunit_config",
+#        ":kunit_parser",
+#        ":kunit_kernel",
+#        ":kunit_json",
+#        ":kunit",
+#    ],
+#)
```

Comment:
@cbrune This test expects data in a `test_data` directory that doesn't seem to exist in the repo - do you have this testdata? Can we commit it so that the test can run?

Reply:
I don't know anything about this test.